### ISSUE TITLE
 No module named 'nnunet'

### ISSUE THREAD

**Issue by MVP-BOY** [MIC-DKFZ/nnUNet#109]
Hi FabianIsensee:
     Thank you for your contribution.When I was doing Experiment Planning and Preprocessing, I was prompted with No module named 'nnunet'.  I installed it according to the Installation requirements. When I am prompted with "Successfully installed nnunet" after pip install-e.  However,When I was doing Experiment Planning and Preprocessing, I was prompted with No module named 'nnunet'.I don't know how to solve it. I hope to get your help.
Best wishes

**Comment by yl255** [comment]
Hi,
Maybe you can create a newfile named 'xxx.pth' under the PythonDir/Lib/site-packages.
And add the nnUNet root path (like: '/deep/nnUNet')to the file.

**Comment by FightingZhen** [comment]
Hi,
You just need to go back to the parent dir of current path and run the code with nnunet/xxxxx/xxxx.py

**Comment by RuiruiKang** [comment]
Is this problem solved?
I have the same problem now

**Comment by FabianIsensee** [comment]
This is not an issue. You just need to cd to some other directory after
running pip install.
Best,
Fabian

**Comment by mogliloew** [comment]
I am running a program from the ISLES22. I installed the nnunet but i get the following error code when i run the program. 

"500_Ischemic_Stroke_Test.py", line 35, in <module>
    from nnunet.dataset_conversion.utils import generate_dataset_json
ModuleNotFoundError: No module named 'nnunet'
nnunet_launcher.sh: line 15: nnUNet_change_trainer_class: command not found
nnunet_launcher.sh: line 19: nnUNet_predict: command not found
nnunet_launcher.sh: line 31: nnUNet_predict: command not found
nnunet_launcher.sh: line 43: nnUNet_predict: command not found
nnunet_launcher.sh: line 55: nnUNet_predict: command not found
nnunet_launcher.sh: line 67: nnUNet_predict: command not found
"

Tried the information linked here but it didnt fix my problem. Thank you in advance for any help
